src/OrchardCore.Modules/OrchardCore.Users/Localization/ViewModel/UserLocalizationViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace OrchardCore.Users.Localization.ViewModels;
+
+public class UserLocalizationViewModel
+{
+    public string SelectedCulture { get; set; }
+
+    [BindNever]
+    public IEnumerable<SelectListItem> CultureList { get; set; } = new List<SelectListItem>();
```

Comment:
```suggestion
    public List<SelectListItem> CultureList { get; set; } = [];
```

We are not trying to abstract the actual collection so using the concrete type is better.